client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/RestaurantGateway.kt

```diff
@@ -33,15 +36,16 @@ class RestaurantGateway(private val client: HttpClient) : BaseGateway(), IRestau
             rating = "4.5".toDouble(),
             priceLevel = 4,
             workingHours = restaurant.workingHours
-        )
+        )//Todo: implement endpoint deleteRestaurants
     }
 
     override suspend fun getCuisines(): List<String> {
-        return listOf("Italian", "Chinese", "Mexican", "American", "Indian", "Japanese", "Thai")
+       return tryToExecute<ServerResponse<List<CuisineDto>>>(client) {
+            get(urlString = "/cuisines")
+        }.value?.map { it.name }?: emptyList()
```

Comment:
why we return empty list if it's null, i think if it's null then something went wrong so we want to show an error message

Reply:
@Karrar-Mohammed should we present to user that we have an error in the backend???
or we can show him an empty list only!
I think we can manage this case by depending on the backend response whether it was an error or just a blank list!
is that correct?
@Asiasama710